patches/@microsoft__api-extractor@7.47.8.patch

```diff
@@ -1,33 +1,28 @@
 diff --git a/lib/enhancers/ValidationEnhancer.js b/lib/enhancers/ValidationEnhancer.js
-index cdb0b22ed2e06592ea1c5f9dd3d18ae2c51b2484..a16f260579c3a696ea0692f4db8d74b19aa120fb 100644
+index cdb0b22ed2e06592ea1c5f9dd3d18ae2c51b2484..202d2f0b44dad1206fff946e0275bba1e6691ade 100644
 --- a/lib/enhancers/ValidationEnhancer.js
 +++ b/lib/enhancers/ValidationEnhancer.js
-@@ -187,15 +187,14 @@ class ValidationEnhancer {
+@@ -187,7 +187,8 @@ class ValidationEnhancer {
              else {
                  continue;
              }
 -            if (collectorEntity && collectorEntity.consumable) {
--                if (api_extractor_model_1.ReleaseTag.compare(declarationReleaseTag, referencedReleaseTag) > 0) {
--                    collector.messageRouter.addAnalyzerIssue(ExtractorMessageId_1.ExtractorMessageId.IncompatibleReleaseTags, `The symbol "${astDeclaration.astSymbol.localName}"` +
--                        ` is marked as ${api_extractor_model_1.ReleaseTag.getTagName(declarationReleaseTag)},` +
--                        ` but its signature references "${localName}"` +
--                        ` which is marked as ${api_extractor_model_1.ReleaseTag.getTagName(referencedReleaseTag)}`, astDeclaration);
--                }
--            }
--            else {
 +            // BUG MITIGATION: Always check release tag compatibility.
-+            if (api_extractor_model_1.ReleaseTag.compare(declarationReleaseTag, referencedReleaseTag) > 0) {
-+                collector.messageRouter.addAnalyzerIssue(ExtractorMessageId_1.ExtractorMessageId.IncompatibleReleaseTags, `The symbol "${astDeclaration.astSymbol.localName}"` +
-+                    ` is marked as ${api_extractor_model_1.ReleaseTag.getTagName(declarationReleaseTag)},` +
-+                    ` but its signature references "${localName}"` +
-+                    ` which is marked as ${api_extractor_model_1.ReleaseTag.getTagName(referencedReleaseTag)}`, astDeclaration);
-+            }
++            {
```

Comment:
I kept the braces and indent so the diff would be smaller, but the functional change is the same as before.